### PR TITLE
[FIX] hr_timesheet_attendance: fix the timesheet attendance report

### PR DESCRIPTION
**Pull request by kcv-odoo**
Attendance report is calculated from the timesheets of all the companies
when multi-company is enabled but only one company is allowed. This commit fixes it.

Steps to reproduce:
 - Create two companies
 - Create an employee with a user allowed in both companies
 - Create timesheets in both companies for that employee
 - Allow only one of the two companies
 - Check the Timesheet Attendance report -> timesheets from both companies are counted

This issue arrived from this commit:
https://github.com/odoo/odoo/commit/c69fe38b946089919c8939bbd9d1adc2602ddd2c

task-2782768

### CODE REVIEW

**Comment by MissingNoShiny** [comment]
Hello, thanks for the fix. :slightly_smiling_face: 
The code looks good to me, but the steps to reproduce in the description were a bit confusing.
Could you maybe rephrase your commit message and PR description like below?
```
[FIX] hr_timesheet_attendance: fix the timesheet attendance report

Attendance report is calculated from the timesheets of all the companies
when multi-company is enabled but only one company is allowed. This commit fixes it.

Steps to reproduce:
 - Create two companies
 - Create an employee with a user allowed in both companies
 - Create timesheets in both companies for that employee
 - Allow only one of the two companies
 - Check the Timesheet Attendance report -> timesheets from both companies are counted

This issue arrived from this commit:
https://github.com/odoo/odoo/commit/c69fe38b946089919c8939bbd9d1adc2602ddd2c

task-2782768
```
Feel free to change what I've written if I misunderstood what you meant while rephrasing.
You can green your task once you are done. :slightly_smiling_face:

**Comment by kcv-odoo** [comment]
> Hello, thanks for the fix. slightly_smiling_face The code looks good to me, but the steps to reproduce in the description were a bit confusing. Could you maybe rephrase your commit message and PR description like below?
> 
> ```
> [FIX] hr_timesheet_attendance: fix the timesheet attendance report
> 
> Attendance report is calculated from the timesheets of all the companies
> when multi-company is enabled but only one company is allowed. This commit fixes it.
> 
> Steps to reproduce:
>  - Create two companies
>  - Create an employee with a user allowed in both companies
>  - Create timesheets in both companies for that employee
>  - Allow only one of the two companies
>  - Check the Timesheet Attendance report -> timesheets from both companies are counted
> 
> This issue arrived from this commit:
> https://github.com/odoo/odoo/commit/c69fe38b946089919c8939bbd9d1adc2602ddd2c
> 
> task-2782768
> ```
> 
> Feel free to change what I've written if I misunderstood what you meant while rephrasing. You can green your task once you are done. slightly_smiling_face

Done! Thanks :slightly_smiling_face:

**Review by IT-Ideas — APPROVED** [review]
Thanks 🙂 
@robodoo r+
